ios/brave-ios/Sources/DataImporter/Models/DataImportModel.swift

```diff
@@ -112,7 +112,9 @@ class DataImportModel: ObservableObject {
       self.importError = error
 
       if case .failedToImportPasswordsDueToConflict = error {
-        self.importState = .dataConflict
+        Task.delayed(bySeconds: 1.0) { @MainActor in
+          self.importState = .dataConflict
+        }
```

Comment:
You're already in an async method, just do `try await Task.sleep(...)` directly in it